.github/ISSUE_TEMPLATE/doc_improvement.yml

```diff
@@ -0,0 +1,19 @@
+name: Documentation improvement
+description: Create a report to help us improve the documentation. Alternatively you can just open a pull request with the suggested change.
+labels: [Documentation]
+
+body:
+- type: textarea
+  attributes:
+    label: Describe the issue linked to the documentation
+    description: >
+      Tell us about the confusion introduced in the documentation.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Suggest a potential alternative/fix
+    description: >
+      Tell us how we could improve the documentation in this regard.
+  validations:
+    required: true
```

Comment:
I think it would would be interesting to allow this kind of documentation feedback even if the reporter does not know how to improve the doc themselves.

```suggestion
```